doc/release/2.11.0.rst

```diff
@@ -0,0 +1,586 @@
+         -- <..process decoded data..>
+    end
+    io:finish(timeout)
+
+Streaming can also be useful to upload a large file to a server or to subscribe to changes in ``etcd`` using the gRPC-JSON gateway.
+The example below demonstrates communication with the ``etcd`` stream interface.
+The request data is written line-by-line, and each line represents an ``etcd`` command.
+
+.. code-block:: lua
+
+    local http_client = require('http.client').new()
+
+    local io = http_client:post('http://localhost:2379/v3/watch', nil, {chunked = true})
+    io:write('{"create_request":{"key":"Zm9v"}}')
+    local res = io:read('\n')
+    print(res)
+    -- <..you can feed more commands here..>
+    io:finish()
+
+
+
+
+.. _linearizable_read:
+
+Linearizable read
+~~~~~~~~~~~~~~~~~
+
+Linearizability of read operations implies that if a response for a write request arrived earlier than a read request was made, this read request should return the results of the write request.
+Tarantool 2.11 introduces the new ``linearizable`` isolation level for :ref:`box.begin() <box-begin>`:
+
+.. code-block:: lua
+
+    box.begin({txn_isolation = 'linearizable', timeout = 10})
+    box.space.my_space:select({1})
+    box.commit()
+
+When called with ``linearizable``, ``box.begin()`` yields until the instance receives enough data from remote peers to be sure that the transaction is linearizable.
+
+There are several prerequisites for linearizable transactions:
+
+*   Linearizable transactions may only perform requests to synchronous, local, or temporary memtx spaces.
+*   Starting a linearizable transaction requires :ref:`box.cfg.memtx_use_mvcc_engine <cfg_basic-memtx_use_mvcc_engine>` to be set to ``true``.
+*   The node is the replication source for at least ``N - Q + 1`` remote replicas.
+    Here ``N`` is the count of registered nodes in the cluster and ``Q`` is :ref:`replication_synchro_quorum <cfg_replication-replication_synchro_quorum>`.
+    So, for example, you can't perform a linearizable transaction on anonymous replicas.
+
+
+
+.. _sequential_scanning_sql:
+
+Explicit sequential scanning in SQL
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Tarantool is primarily designed for OLTP workloads.
+This means that data reads are supposed to be relatively small.
+However, a suboptimal SQL query can cause a heavy load on a database.
+
+The new ``sql_seq_scan`` :ref:`session setting <sql_set>` is added to explicitly cancel full table scanning.
+The request below should fail with the ``Scanning is not allowed for 'T'`` error:
+
+.. code-block:: sql
+
+    SET SESSION "sql_seq_scan" = false;
+    SELECT a FROM t WHERE a + 1 > 10;
+
+
+To enable table scanning explicitly, use the new ``SEQSCAN`` keyword:
+
+.. code-block:: sql
+
+    SET SESSION "sql_seq_scan" = false;
+    SELECT a FROM SEQSCAN t WHERE a + 1 > 10;
+
+Starting with v3.0, ``SEQSCAN`` will be required for scanning queries with the ability to disable the check using the ``sql_seq_scan`` session setting.
```

Comment:
Pls avoid future promises with exact dates or versions.

Reply:
@Totktonada thanks for clarification!